security/subsystem/src/main/java/org/jboss/as/security/SecurityExtension.java

```diff
@@ -256,6 +291,31 @@ private void registerTransformers_1_2_0(SubsystemRegistration subsystemRegistrat
         TransformationDescription.Tools.register(builder.build(), subsystemRegistration, ModelVersion.create(1, 2, 0));
     }
 
+    private void registerTransformers_1_3_0(SubsystemRegistration subsystemRegistration) {
+        // earlier versions of the model didn't have the export-elytron-realm attribute, but the subsystem behavior is
+        // fully compatible if this attribute is left undefined or if its set to false.
+        ResourceTransformationDescriptionBuilder builder = TransformationDescriptionBuilder.Factory.createSubsystemInstance();
+        ResourceTransformationDescriptionBuilder securityDomain = builder.addChildResource(SECURITY_DOMAIN_PATH);
+        securityDomain.getAttributeBuilder().setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(false, true, new ModelNode(false)),
+                SecurityDomainResourceDefinition.EXPORT_ELYTRON_REALM)
+                .addRejectCheck(new RejectAttributeChecker.DefaultRejectAttributeChecker() {
+
+                    @Override
+                    public String getRejectionLogMessage(final Map<String, ModelNode> attributes) {
+                        return SecurityLogger.ROOT_LOGGER.rejectedAttributeMustUndefinedOrFalse();
+                    }
+
+                    @Override
+                    protected boolean rejectAttribute(final PathAddress address, final String attributeName, final ModelNode attributeValue, final TransformationContext context) {
+                        return attributeValue.asBoolean();
+                    }
+                }, SecurityDomainResourceDefinition.EXPORT_ELYTRON_REALM)
```

Comment:
You have the discard above which will discard 'false', so you can reject always. This looks like the scenario discussed here:

https://docs.jboss.org/author/display/WFLY10/Domain+Mode+Subsystem+Transformers#DomainModeSubsystemTransformers-Defaultvalueoftheattributeisthesameaslegacyimpliedbehaviour